djstripe/__init__.py

```diff
@@ -16,8 +14,14 @@
 __license__ = "License :: OSI Approved :: BSD License"
 __copyright__ = "Copyright 2015 Daniel Greenfeld"
 
-if get_django_version() <= '1.6.x':
-    msg = "dj-stripe deprecation notice: Django 1.6 and lower are no longer\n" \
-        "supported. Please upgrade to Django 1.7 or higher.\n" \
-        "Reference: https://github.com/pydanny/dj-stripe/issues/173"
-    warnings.warn(msg)
+try:
```

Comment:
I'm fairly sure [this commit](https://github.com/pydanny/dj-stripe/commit/de62268d60053af6dbdba3304ce964e8c4377969) fixed this issue.